src/blocks/modal.css

```diff
@@ -98,6 +98,12 @@
   opacity: 1;
 }
 
+.modal__close-btn_type_delete {
+  top: -12px;
+  right: -36px;
+  opacity: 1;
+}
```

Comment:
The `close icon` of the **confirmation** modal is not correctly positioned https://snipboard.io/jqCX8B.jpg